### DISCUSSION TITLE
Uncommited Change Highlight

### DISCUSSION THREAD

**Discussion by aryankarim** [Kong/insomnia#7237] [Ideas]
We can have a file highlight just like vscode. Plus individual file "discard change button" in the requests menu.

Just like shown below:

![fff](https://github.com/Kong/insomnia/assets/58115075/e4102497-b668-47dc-948d-5a689c232cb1)

**Comment by subnetmarco** [comment]
This is a very nice suggestion. We are about to ship a new diff viewer by end of month, but the ability to see the change in the sidebar itself would be very nice indeed. Let's see what we can do here.
